[en]update insight-userguide docs (#5200)
DaoCloud/DaoCloud-docs: docs/en/docs/insight/user-guide/trace/service.md

| ------ | ----------- |
| Throughput Rate | The number of requests processed within a unit of time. |
| Error Rate | The ratio of erroneous requests to the total number of requests within the specified time range. |

> | P50 Request Latency | 50% of all request response times are less than or equal to this value. |
> | P95 Request Latency | 95% of all request response times are less than or equal to this value. |
> | P99 Request Latency | 99% of all request response times are less than or equal to this value. |

Reviewer: ```suggestion
| P50 Request Latency | The response time within which 50% of requests complete. |
| P95 Request Latency | The response time within which 95% of requests complete. |
| P99 Request Latency | The response time within which 99% of requests complete. |
```
这样会不会更清晰点
